cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/0-intro.html

```diff
@@ -0,0 +1,22 @@
+<div class="college-costs_intro-segment">
+    {% for block in page.header -%}
+        {% if 'hero' in block.block_type %}
+            {{ render_stream_child(block) }}
+        {% endif %}
+    {%- endfor %}
+    <div class="wrapper content_wrapper">
```

Comment:
Is there a `content` in a parent div container somewhere?

Reply:
Yes, the parent `content` container is on L19 of the `college-costs.html` template.